filewriter_v2/README.md

```diff
@@ -0,0 +1,399 @@
+# FileWriter V2
+
+---
+$\color{red}\text{\large{NOTE: FileWriter V2 is currently in a preview state and will change in the future.}}$
+
+Feedback is very welcome!
+
+---
+
+FileWriter writes image data and metadata to [HDF5](https://www.hdfgroup.org/solutions/hdf5/) files. Currently, there are two different formats available. V1 is a legacy format that is loosely based on [version 1.4](https://github.com/nexusformat/definitions/releases/tag/NXmx-1.4) of the _NXmx application definition_ of the [NeXus] format. V2 follows the [_NXmx application definition_](https://manual.nexusformat.org/classes/applications/NXmx.html) of NeXus Version 2022.07, with the addition of multi-channel data support[^1].
+
+| Version | Configuration | NeXus/NXmx Version | Multi-Channel Data |
+|----|----|----|----|
+| V1 | SIMPLON | [NeXus NXmx 1.4 (2016)](https://github.com/nexusformat/definitions/blob/main/legacy_docs/nexus-v3.2-2016-10-10-ad046b5.pdf) | No |
+| V2 | SIMPLON | [NeXus 2022.07 NXmx](https://github.com/nexusformat/definitions/blob/v2022.07/applications/NXmx.nxdl.xml) | Yes |
+
+[^1]: This is not part of the NeXus standard yet, a detailed PR will be opened when this data format has been approved by the community. The general idea of this representation has already been discussed and approved by the NIAC though, see this [issue](https://github.com/nexusformat/definitions/issues/940).
+
+
+## FileWriter V2 Configuration
+
+FileWriter V2 can be enabled and configured using the SIMPLON API.
+
+### SIMPLON Configuration Parameters
+
+The configuration interface is accessible at `http://<ADDRESS_OF_DCU>/filewriter/api/1.8.0/config`.
+
+| Parameter | JSON Type | Default | Description |
+|-----------|------|---------|-------------|
+| `mode` | string | `"disabled"` | Enabled state of the FileWriter, either `"enabled"` or `"disabled"`. |
+| `compression_enabled` | boolean | `true` | Optionally disable compression of image data. |
+| `image_nr_start` | number | `1` | Unsigned integer defining the first image number on the axis `/entry/image_id`. |
+| `name_pattern` | string | `"series_$id"` | Base name of the files. `$id` will be replaced by the series number, i.e. using the default, the generated files will have the names:<br> `series_<series_number>_master.h5`<br> `series_<series_number>_data_<file_number>.h5`|
```

Comment:
Replace uses of "series_number" with "series_id".

Reply:
done in [3180a11](https://github.com/soph-dec/documentation/pull/1/commits/3180a11e6d8ecbd3ffed1ada11e2a746b47eef15)